auv_navigation/auv_mapping/src/process_tracker_with_cloud.cpp

```diff
@@ -0,0 +1,635 @@
+/**
+ * auv_mapping
+ * ROS node for processing PointCloud2 data with YOLO detections
+ * Inspired by ultralytics_ros tracker_with_cloud_node
+ */
+
+#include <geometry_msgs/TransformStamped.h>
+#include <image_geometry/pinhole_camera_model.h>
+#include <message_filters/subscriber.h>
+#include <message_filters/sync_policies/approximate_time.h>
+#include <message_filters/synchronizer.h>
+#include <ros/ros.h>
+#include <sensor_msgs/CameraInfo.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <tf2/LinearMath/Matrix3x3.h>
+#include <tf2_ros/buffer.h>
+#include <tf2_ros/transform_broadcaster.h>
+#include <tf2_ros/transform_listener.h>
+#include <ultralytics_ros/YoloResult.h>  // vision_msgs/Detection2DArray yerine
+#include <vision_msgs/Detection3DArray.h>
+#include <visualization_msgs/MarkerArray.h>
+
+// PCL Kütüphaneleri
+#include <pcl/common/centroid.h>
+#include <pcl/common/common.h>  // getMinMax3D için gerekli
```

Comment:
You have written that you no longer needed the getMinMax3D so :

```suggestion
#include <pcl/common/common.h>
```